mods/voidmulti.sc2mod/base.sc2data/gamedata/abildata.xml

```diff
@@ -1525,5 +1691,6 @@
     </CAbilResearch>
     <CAbilMove id="move">
         <AbilSetId value="Move"/>
+        <FollowAcquireRange value="2"/>
```

Comment:
All Units: Follow acquire range reduced from 5 to 2 (Units will switch to attacking from moving when within this range of the follow target)

=> Accidental move commands on your own units while retreating won't cause your army to fight around that unit in a wider area anymore. The area is now much smaller

Could this render melee units unable to attack ranged units while following another unit/structure?